app/views/cases/shared/_closed_case_details.html.slim

```diff
@@ -67,3 +67,8 @@ tbody.response-details
       th = t('common.case.outcome')
       td = case_details.pretty_ico_decision
 
+  - if case_details.retention_schedule.present? && FeatureSet.branston_retention_scheduling.enabled?
+    tr.planned-erasure-date
+      th = "Planned erasure date"
```

Comment:
Super minor thing! String is not localised (was looking at the other strings in this file!)

Reply:
Thanks this is a good point - there is another ticket to update the wording away from "erasure" to destroy, so I just left this as is and will localise it when I do the update to the wording across the app. Was thinking that the key would probably include the word "erasure" if I did it now - probably just overthinking a minor thing though 🙂